Set-LicenseEnvironmentVariable.ps1

```diff
@@ -7,17 +7,16 @@ param(
     [switch]$PersistForCurrentUser
 )
 
-$licenseFileBytes = [System.IO.File]::ReadAllBytes($Path)
+$licenseFileStream = [System.IO.File]::OpenRead($Path);
```

Comment:
You should close the `FileStream` opened here once you're done with it.

Reply:
I don't think that's needed, as the GZipStream takes care of that when it is being closed. It's why the third constructor argument is set to $false. [Check here for more info.](https://docs.microsoft.com/en-us/dotnet/api/system.io.compression.gzipstream.-ctor?view=netframework-4.8#System_IO_Compression_GZipStream__ctor_System_IO_Stream_System_IO_Compression_CompressionMode_System_Boolean_)

Reply:
I completely missed that ctor argument - sorry about that!